packages/connect-explorer/src/pages/index.mdx

```diff
@@ -321,6 +245,64 @@ export const ExampleHeading = styled.h3`
             -   [electron in renderer layer with popup](https://github.com/trezor/trezor-suite/tree/develop/packages/connect-examples/electron-renderer-with-popup)
         </ExamplesAside>
     </SdkContainer>
+    <CollapsibleBox heading="Quick start manual" margin={{ top: spacings.md }}>
+            <Steps>
+                ### Installation of the package
+
+                ```javascript
+                npm install @trezor/connect-web
+                # or
+                yarn add @trezor/connect-web
+                ```
+
+                or include the library as inline script:
+
+                ```html
+                <script src="https://connect.trezor.io/9/trezor-connect.js"></script>
+                ```
+
+                ### Import from the package
+
+                ```javascript
+                import TrezorConnect from '@trezor/connect-web';
+                ```
+
+                or use inline:
+
+                ```javascript
+                var TrezorConnect = window.TrezorConnect;
+                ```
+
+                ### Initialization of the project
+
+                ```javascript
+                TrezorConnect.init({
+                    lazyLoad: true, // this param prevents iframe injection until
+                    // TrezorConnect.method is called
+                    manifest: {
+                        email: 'developer@xyz.com',
+                        appUrl: 'http://your.application.com',
+                    },
+                });
+                ```
+
+                Trezor Connect Manifest requires that you, as a Trezor Connect integrator, share your email and application URL with us. This provides us with the ability to reach you in case of any required maintenance. This subscription is mandatory.
+
+                If you need more customization, refer to [init method documentation](./methods/other/init)
+
+                ### How to use?
```

Comment:
I wouldn't include the "How to use?" section in the `<Steps> </Steps>` as it is not really needed for a setup. WDYT?

Reply:
I would leave it for consitency

Reply:
Ok, but can we put it out of the <Steps> and leave it as a note, or something like that…

Reply:
It's not a problem to change it, but personally I think it looks more coherent as a step, because the steps are offset due to the numbering, so it's not aligned when it's outside.
![Screenshot 2024-06-18 at 11 13 52](https://github.com/trezor/trezor-suite/assets/8833813/ccf8ade2-1a30-4177-a416-dfaa05ff86c0)

Reply:
Ok, let's then leave it as it is in `<Step>` component